pkgs/development/libraries/libexif/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "0mzndakdi816zcs13z7yzp7hj031p2dcyfq2p391r63d9z21jmy1";
   };
 
+  patches = [
```

Comment:
I'm able to apply `dee5dea213f4eafea12a2bd4999d7de526b02cf2` to `libexif-0_6_22-release`.
`9266d14b5ca4e29b970fa03272318e5f99386e06` doesn't apply out of the box, but should be solvable with something like this:

```
fetchpatch {
  url = ...;
  excludes = "NEWS";
  sha256 = ...;
}
```

I'd prefer to have these commits fetched via fetchpatch, so we don't have to maintain them in tree.

Reply:
> Had to rebase the commits, because they don't cleanly apply without picking other stuff, so I added them as patches to the repo.

Reply:
Thanks for the pointer to `excludes`, I did not know about this.